github_pr.py

```diff
@@ -369,8 +398,23 @@ def main():
 Delete a PR
 
     github-pr delete -r dataxu/test_repo -n 17
+
+Check conditional status checks
+    !!!This check only looks at comments AFTER the latest commit, to validate that the
+    code has been most recently peer reviewed.!!!
```

Comment:
`most recent code (most recent git sha pushed to the PR) has been peer reviewed`